test/e2e/specs/infrastructure/notification__actions.ts

```diff
@@ -0,0 +1,127 @@
+/**
+ * @group calypso-pr
+ */
+
+import {
+	DataHelper,
+	NavbarComponent,
+	NotificationsComponent,
+	RestAPIClient,
+	ReaderResponse,
+	SecretsManager,
+	TestAccount,
+	NewCommentResponse,
+} from '@automattic/calypso-e2e';
+import { Page, Browser } from 'playwright';
+
+declare const browser: Browser;
+
+describe( DataHelper.createSuiteTitle( 'Notifications' ), function () {
+	const commentingUser = 'commentingUser';
+	const notificationsUser = 'notificationsUser';
+	const comment = DataHelper.getRandomPhrase() + ' notification-actions-spec';
+
+	let notificationsComponent: NotificationsComponent;
+	let restAPIClient: RestAPIClient;
+	let siteID: number;
+	let commentID: number;
+
+	describe( `Leave a comment as ${ commentingUser }`, function () {
+		let postID: number;
+
+		beforeAll( async function () {
+			restAPIClient = new RestAPIClient( SecretsManager.secrets.testAccounts.commentingUser );
+		} );
+
+		it( 'Get the latest post in reader stream', async function () {
+			const response: ReaderResponse = await restAPIClient.getReaderFeed();
+			siteID = response.posts[ 0 ].site_ID;
+			postID = response.posts[ 0 ].ID;
+		} );
+
+		it( 'Leave a comment on the post', async function () {
+			const response: NewCommentResponse = await restAPIClient.createComment(
+				siteID,
+				postID,
+				comment
+			);
+			commentID = response.ID;
+		} );
+	} );
+
+	describe( `View notification as ${ notificationsUser }`, function () {
+		let page: Page;
+
+		beforeAll( async function () {
+			page = await browser.newPage();
+			const testAccount = new TestAccount( notificationsUser );
+			await testAccount.authenticate( page );
+		} );
```

Comment:
**style:non-blocking**: I'd say this should be moved out to the top level `beforeAll`, since the setup contained here isn't particular to the `View notification` block.

No harm leaving it here either, hence it's just a non-blocking comment.

Reply:
Sounds good. I've moved it to the top level one in the new commit. Thank you!